packages/components/atoms/f-link/src/components/Link.vue

```diff
@@ -1,45 +1,127 @@
 <template>
-    <div
-        :class="$style['o-link']"
-        data-test-id="link"
-    >
-        {{ copy.text }}
-    </div>
+    <a
+        :class="[
+            $style['o-link'],
+            { [$style['o-link--bold']]: isBold },
+            { [$style['o-link--noDecoration']]: !hasTextDecoration },
+            { [$style['o-link--full']]: isFullWidth }
+        ]"
+        :target="target"
+        :aria-label="ariaLabel"
+        :rel="rel"
+        v-bind="$attrs"
+    >{{ linkText }}</a>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
 import tenantConfigs from '../tenants';
 
 export default {
     name: 'VLink',
-    components: {},
+
+    mixins: [VueGlobalisationMixin],
+
     props: {
-        locale: {
+        linkText: {
+            type: String,
+            required: true
+        },
+
+        url: {
```

Comment:
Url is not in use anymore so this can be deleted

Reply:
Thought I'd got rid of this already. Thanks 🙂